### PR TITLE
Add license file

### PR DESCRIPTION
**Pull request by marcelotrevisani**
Add LICENSE file to the project

Please review @fafhrd91

Related to https://github.com/PyO3/setuptools-rust/issues/40

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2017-2018 Nikolay Kim & contributors
```
Comment (messense) [inline LICENSE]: I think it's better to add `MIT License` in the first line.

Comment (marcelotrevisani) [reply LICENSE]: Got it

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017-2018 Nikolay Kim & contributors
```
Comment (fafhrd91) [inline LICENSE]: Lets use PyO3 project and contributors

Comment (marcelotrevisani) [reply LICENSE]: Got it! I'm going to change

**Pull request merged**